requirements.csv

```diff
@@ -0,0 +1,19 @@
+vendor,product
```

Comment:
Since these files risk getting out of sync with the `.txt` versions, we should either have a lookup where we map the `.txt` to the vendor package pair, and make sure that there is a mapping for each, or else `raise` an exception that we need to update the mapping. OR we should have a test which checks that all the entries in the `.txt` file also appear in the `.csv` file (for each file respectively).